new-dti-website-redesign/src/components/FAQAccordion.tsx

```diff
@@ -0,0 +1,54 @@
+'use client';
+
+import { ReactNode } from 'react';
+
+// Props type for the FAQAccordion component
+type FAQAccordionProps = {
+  header: string;
+  children: ReactNode;
+};
+
+/**
+ * Creates a collapsible accordion component using Radix UI
+ *
+ * @param header - The text displayed in the accordion trigger/header
+ * @param children - The content shown when the accordion is opened
+ */
+export default function FAQAccordion({ header, children }: FAQAccordionProps) {
+  return (
+    <details className="group border border-border-1">
+      <summary
+        className="
+          list-none p-8
+          focus-visible:outline focus-visible:outline-[2px] focus-visible:outline-offset-[3px] focus-visible:outline-[var(--foreground-1)] focus-visible:relative 
```

Comment:
```suggestion
         focusState focus-visible:relative 
```

we can actually replace much of these focus state props with `focusState`. It's defined in globals.css on line 187 and applies the same functionality